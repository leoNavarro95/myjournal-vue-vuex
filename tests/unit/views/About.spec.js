
import { shallowMount } from '@vue/test-utils'

import About from '@/views/About';

describe( ' About View component ', () => {

    test( ' Debe renderizarse correctamente --/match con el snapshot ', () => {
        const wrapper = shallowMount( About )

        expect( wrapper.html() ).toMatchSnapshot()
    })
    
})