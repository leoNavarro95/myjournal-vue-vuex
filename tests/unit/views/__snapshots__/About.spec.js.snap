// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` About View component   Debe renderizarse correctamente --/match con el snapshot  1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
